packages/frontend/src/redux/slices/tokenFiatValues/index.js

```diff
@@ -85,11 +85,11 @@ export const selectNearTokenFiatValueUSD = createSelector(selectNearTokenFiatDat
 
 export const selectUSDNTokenFiatData = createSelector(
     selectAllTokenFiatValues,
-    ({ tokens }) => tokens.tether || {}
+    ({ tokens }) => tokens.usn || {}
```

Comment:
USN values were being implied via tether stablecoin values, changed to fetch usn values from coingecko in testnet and usn values from ref-finance in mainnet